mlir/lib/Dialect/Linalg/TransformOps/LinalgTransformOps.cpp

```diff
@@ -1126,7 +1126,7 @@ DiagnosedSilenceableFailure transform::LowerUnPackOp::applyToOne(
   FailureOr<LowerUnPackOpResult> res = lowerUnPack(rewriter, target);
   if (failed(res)) {
     return mlir::emitSilenceableFailure(target->getLoc())
```

Comment:
The error is usually attached to the transform op. You can add a note that points to the payload op.

E.g.:
```c++
DiagnosedSilenceableFailure diag = emitSilenceableError()
                                   << "cannot lower to transpose + collapse + extract";
diag.attachNote(target->getLoc()) << "target payload op";
return diag;
```

Reply:
thanks, added.